velero-plugins/imagestream/backup.go

```diff
@@ -41,8 +41,15 @@ func (p *BackupPlugin) Execute(item runtime.Unstructured, backup *v1.Backup) (ru
 		annotations = make(map[string]string)
 	}
 
-	skipImages := annotations[common.SkipImages]
-	if len(skipImages) != 0 {
+	if val, ok := backup.Annotations[common.SkipImageCopy]; ok {
+		annotations[common.SkipImageCopy] = val
+		p.Log.Info("[is-backup] Propagating backup annotation to individual ImageStream")
+	} else {
+		p.Log.Info("[is-backup] Backup annotation not specified")
+	}
+
+	skipImages := annotations[common.SkipImageCopy]
+	if len(skipImages) != 0 && skipImages == "true" {
 		p.Log.Info("Not running in OADP/CAM context, skipping copy of image.")
```

Comment:
We should probably change the Info log here. Previously the skipImages part was only used when a backup/restore was run outside of MTC/OADP. Now we're also using this from MTC with skip image copy set. Maybe something like this instead:
"Image copy is excluded for backup; skipping image copy"

Reply:
@sseago This change has been made.